src/components/crownConfig.vue

```diff
@@ -0,0 +1,273 @@
+<template>
+  <div class="crown-config" :style="style" v-if="showCrown">
+    <sequence-flow-button
+      @click="addSequence()"
+      class="crown-config__icon"
+      :title="$t('Sequence Flow')"
+    />
+    <message-flow-button
+      @click="addMessageFlow()"
+      class="crown-config__icon"
+      :title="$t('Message Flow')"
+    />
+    <delete-button
+      @click="removeShape()"
+      class="crown-config__icon"
+      :title="$t('Delete')"
+    />
+  </div>
+</template>
+
+<script>
+import DeleteButton from '@/components/deleteButton';
+import MessageFlowButton from '@/components/messageFlowButton';
+import SequenceFlowButton from '@/components/sequenceFlowButton';
+import pull from 'lodash/pull';
+
+export default {
+  components: {
+    DeleteButton,
+    MessageFlowButton,
+    SequenceFlowButton,
+  },
+  props: [
+    'highlighted',
+    'paper',
+    'graph',
+    'shape',
+    'node',
+    'nodeRegistry',
+    'moddle',
+    'planeElements',
+    'processNode',
+    'collaboration',
+  ],
+  watch: {
+    highlighted() {
+      this.showCrown = this.highlighted;
+    },
+    shape() {
+      if (this.highlighted) {
+        this.showCrown = true;
+        this.configurePoolLane();
+        this.setUpCrownConfig();
+      }
+    },
+  },
+  data() {
+    return {
+      showCrown: false,
+      defaultHighlighter: {
+        name: 'stroke',
+        options: {
+          attrs: {
+            stroke: '#feb663',
+            'stroke-width': 3,
+          },
+        },
+      },
+      /* allowSetNodePosition is used to prevent setting a node position outside of a pool */
```

Comment:
Since this seems to be moved to `highlightConfig`, is it still needed here?

Reply:
There was logic for both highlighting and saving the position in the original setUpCrownConfig. I ended up having one in highlightConfig and crownConfig. I believe this is needed to position it correctly after undo/redo. Which didn't seem to be highlight related.

Reply:
I think I highlighted an extra line here; I was referring to removing `defaultHighlighter` since it's not used in this file, but appears to be moved over to the new highlightConfig mixin.

Reply:
Whew. Thanks. That can go.